sdk/language-support/java/json/BUILD.bazel

```diff
@@ -32,6 +32,7 @@ da_scala_library(
         "//daml-lf/data",
         "//daml-lf/transaction",
         "//ledger-service/lf-value-json",
+        "@maven//:com_google_protobuf_protobuf_java",
```

Comment:
Added so that the ledger codegen may access the protobuf `ByteString` data structure